appium/tests/specs/live/inbox/CheckMessageProcessingErrors.spec.ts

```diff
@@ -66,16 +70,28 @@ describe('INBOX: ', () => {
     await EmailScreen.checkOpenedEmail(wrongChecksumEmail, wrongChecksumSubject, wrongChecksumText);
     await EmailScreen.checkEncryptionBadge(decryptErrorBadgeText);
 
-    // Checking integrity protected message BUG:https://github.com/FlowCrypt/flowcrypt-ios/issues/1144
+    // Checking integrity protected message
+    await EmailScreen.clickBackButton();
+    await SearchScreen.clickBackButton();
+    await MailFolderScreen.checkInboxScreen();
+    await MailFolderScreen.clickSearchButton();
 
-    // await EmailScreen.clickBackButton();
-    // await SearchScreen.clickBackButton();
-    // await MailFolderScreen.checkInboxScreen();
+    await SearchScreen.searchAndClickEmailBySubject(notIntegrityProtectedSubject);
+    await MailFolderScreen.clickOnEmailBySubject(notIntegrityProtectedSubject);
+    await EmailScreen.checkOpenedEmail(notIntegrityProtectedEmail, notIntegrityProtectedSubject, notIntegrityProtectedText);
+    await EmailScreen.checkEncryptionBadge(decryptErrorBadgeText);
+
+    // Checking key mismatch unexpectedly produces a modal message
```

Comment:
We're checking that app doesn't show modal and renders message body, so comment should be `Checking if message body is rendered with key mismatch`